modules/v2x/v2x_proxy/app/BUILD

```diff
@@ -6,14 +6,55 @@ package(default_visibility = ["//visibility:public"])
 cc_library(
     name = "v2x_proxy_lib",
     srcs = [
-        "main.cc",
         "v2x_proxy.cc",
+        "utils.cc",
+        "main.cc",
+    ],
+    hdrs = [
+        "v2x_proxy.h",
+        "utils.h",
+    ],
+    deps = [
+        "//modules/v2x/v2x_proxy/obu_interface:obu_interface_grpc_impl",
+        "//modules/v2x/v2x_proxy/os_interface:v2x_proxy_os_interface",
+        "//modules/v2x/v2x_proxy/proto_adapter:v2x_proxy_proto_adapter",
+        "//modules/map/hdmap:hdmap_util",
+        "//cyber",
+    ],
+)
+
+cc_binary(
+    name = "v2x_proxy_test",
+    srcs = [
+        "v2x_proxy_test.cc",
+        "utils.cc",
```

Comment:
`cc_test` should be used for `xxx_test.cc` targets.
Again, please split `utils.cc/h` into a separate `cc_library` target.